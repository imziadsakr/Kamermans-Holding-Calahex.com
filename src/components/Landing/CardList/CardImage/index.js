


import React from 'react' ;

import {
    Box
} from "@mui/material" ;

import { makeStyles } from '@mui/styles';

const useStyles = makeStyles(() => ({
    root : {
        "& img" : {
            width : "100%" ,
            borderRadius : "20px"
        }
    }
}));

const CardImage = (props) => {
    
    const { imageUrl } = props ;
    
    const classes = useStyles() ;

    return (
        <Box  className={classes.root}>
            <Box component={"img"} src={imageUrl} />
        </Box>
    )
}

export default CardImage ;