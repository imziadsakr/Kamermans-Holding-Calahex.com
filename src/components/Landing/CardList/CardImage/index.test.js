import React from 'react' ;

import tradingImg from '../../../../assets/trading_img.jpg' ;

import { shallow } from 'enzyme' ;

import CardImage from '.';

describe('CardImage', () => {
    it('should render image correctly with given strings', () => {
    
        let component = shallow(
            <CardImage imageUrl={tradingImg} />
        );
        expect(component).toMatchSnapshot() ;
    })
})
