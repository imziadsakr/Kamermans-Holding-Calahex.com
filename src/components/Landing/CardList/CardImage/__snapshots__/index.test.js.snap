// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardImage should render image correctly with given strings 1`] = `
<ForwardRef(Box)
  className="makeStyles-root-1"
>
  <ForwardRef(Box)
    component="img"
    src="trading_img.jpg"
  />
</ForwardRef(Box)>
`;
